TESTS/mbedmicro-rtos-mbed/threads/main.cpp

```diff
@@ -26,18 +26,8 @@
 #else
 
 #define THREAD_STACK_SIZE 512
-#if defined(__CORTEX_A9)
 #define PARALLEL_THREAD_STACK_SIZE 512
-#elif defined(__CORTEX_M23) || defined(__CORTEX_M33)
-#define PARALLEL_THREAD_STACK_SIZE 512
-#elif defined(TARGET_ARM_FM)
-#define PARALLEL_THREAD_STACK_SIZE 512
-#elif defined(TARGET_CY8CKIT_062_WIFI_BT_PSA)
-#define PARALLEL_THREAD_STACK_SIZE   512
-#else
-#define PARALLEL_THREAD_STACK_SIZE 384
```

Comment:
why do we get `PARALLEL_THREAD_STACK_SIZE` fixed to 512?  I think in some case some small target would not fit final compiled the images. GR_LYCHEE is an A9 target,  maybe we need to make the macro defines as well in the stats_cpu tests

Reply:
We could, but i tried to remove too many #if statements as it was working on STM32F070RB and the size of the stack was reduced to make it pass on this target.

Reply:
> 
> 
> why do we get `PARALLEL_THREAD_STACK_SIZE` fixed to 512? I think in some case some small target would not fit final compiled the images. GR_LYCHEE is an A9 target, maybe we need to make the macro defines as well in the stats_cpu tests

Such targets should be optimized in the way as it was done here:
https://github.com/ARMmbed/mbed-os/pull/5285/commits/f24ac501a817c3d779dfe0edc2dc6a54db2ec91e
https://github.com/ARMmbed/mbed-os/pull/5285/commits/37bfc9e5e39193d4e8973e413515d07ac93d2873